test/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -271,6 +283,33 @@ import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 				done();
 			});
 		});
+		test('updating filter option updates filters', function() {
+			const update = 2;
+			_setSelectedOptions([0, 4, 5]);
```

Comment:
Is there some significance to what these numbers mean? If so, would it be more clear in an enum?

Or if they're just the place in the list then ignore me :)

Reply:
Yup; they're just indices for the options that we want to select for the test. There's nothing notable about them in specific, other than the fact that they cause the test to end up with different amounts of selected options per category.